prototype2/gdgem/NMXConfig.cpp

```
@@ -50,8 +51,7 @@ NMXConfig::NMXConfig(std::string configfile, std::string calibrationfile) {
     root = nlohmann::json::parse(jsonstring);
   }
   catch (...) {
-    XTRACE(INIT, WAR, "Invalid Json file: %s", configfile.c_str());
-    return;
+    XTRACE(INIT, WAR, "Invalid Json file: %s", configfile.c_str());   	
```

did you not want to throw a runtime exception in this case?

---

Also I guess the severity is probably ERR, right?